packages/middleware/src/createServer.ts

```diff
@@ -50,11 +52,24 @@ async function createServer(config: MiddlewareConfig): Promise<Express> {
 
   app.post('/:integrationName/:functionName', async (req: Request, res: Response) => {
     const { integrationName, functionName } = req.params as any as RequestParams;
+
+    if (!integrations[integrationName]) {
+      const errMsg = `"${integrationName}" integration is not configured. Please, check the request path or integration configuration.` +
+        `Request host: ${req?.headers?.host}, origin: ${req.headers.origin}`;
```

Comment:
1. Why do we print host and origin headers? 
2. Why for `.host` we are using optional chaining but for `.origin` we don't?

Reply:
I did this because recently we had problems on the cloud with identifying the source of a malicious request (auth0 integration request). I can get rid of this part if you think it is not required. I would also expect to have it on logs on Nginx proxy, and as far as I know, the only reason we do not have it is that something in the cloud was upgraded and it breaks the logging mechanism. Ok, you no what, I will remove it. Introducing something because something we expect to work is not working does not make sense here.,